chromadb/utils/query_helper.py

```diff
@@ -0,0 +1,135 @@
+from typing import List, Union, Literal, Any, Dict, cast
+
+from chromadb.types import (
+    WhereOperator,
+    LiteralValue,
+    WhereDocument as WhereDocumentType,
+)
+
+
+class Where(object):
```

Comment:
Let's also export this as a top-level import so you can just do from chromadb import WhereBuilder etc.